etcd/assets/configuration/spec.yaml

```diff
@@ -8,7 +8,9 @@ files:
   - template: instances
     options:
       - name: use_preview
-        description: Whether or not to preview the new version of the check supporting only ETCD 3+
```

Comment:
I removed the language of `preview the new version` since this has been available for a few years now, and there could be an updated OpenMetrics-based check in the future.